### PR TITLE
Changes the vxtvul hammer sounds to be more impactful

### PR DESCRIPTION
**Pull request by solwashere**
# Document the changes in your pull request

Changes the vxtvul hammer sounds to sound more impactful and like you are swinging a heavy chunk of metal designed to kill people.

for some reason github doesnt like OGGs 
https://www.mediafire.com/file/hupc4tlz81lo4r1/hammerhitbasic.ogg/file
https://www.mediafire.com/file/gnpf10s0bmj08xo/powerhammerhit.ogg/file

# Spriting

# Wiki Documentation

# Changelog

:cl:  
rscadd: Adds a dedicated hit sound for the unpowered hammer
tweak: changed the charged up attack hit sound
soundadd: adds powerhammerhit.ogg, hammerhitbasic.ogg, and electrichum.ogg (unused for now)

/:cl:

### CODE REVIEW

**Comment by cuackles** [comment]
https://user-images.githubusercontent.com/68789204/231898492-fb31b700-2cfe-49fd-8816-95e4fc0d1df9.mp4


https://user-images.githubusercontent.com/68789204/231898495-301700b2-999c-4893-ba34-2c07c0476b0c.mp4

gonna put em here on mp4 so its easier to hear
also dear god they are loud

**Comment by Moltijoe** [comment]
> gonna put em here on mp4 so its easier to hear also dear god they are loud

i assume the first one is the charged one?
also, is that from something? i vaguely feel like i recognize it

**Comment by cuackles** [comment]
> > gonna put em here on mp4 so its easier to hear also dear god they are loud
> 
> i assume the first one is the charged one? also, is that from something? i vaguely feel like i recognize it

my bet by the style is the sound is fire force anime

**Comment by Moltijoe** [comment]
> my bet by the style is the sound is fire force anime

oh, maybe, i'm sure sol would know though

**Comment by solwashere** [comment]
https://user-images.githubusercontent.com/119529668/231901203-7ef2c96e-7c99-45b6-b8f5-2772970fd8a6.mp4

video demonstration

**Comment by solwashere** [comment]
> > my bet by the style is the sound is fire force anime
> 
> oh, maybe, i'm sure sol would know though

Risk of Rain 2

**Comment by solwashere** [comment]
basic spelling error L

**Comment by Moltijoe** [comment]
> Risk of Rain 2

mithrix sound effect i'm pretty sure

**Comment by Moltijoe** [comment]
> video demonstration

hmm, i feel like the sound varience should probably be removed from the new charged sound effect
it just feels wrong pitched up or down

**Comment by solwashere** [comment]
> > video demonstration
> 
> hmm, i feel like the sound varience should probably be removed from the new charged sound effect it just feels wrong pitched up or down

by that do you mean the explosion sound that plays alongside the current one?

File: code/game/objects/items/twohanded.dm [hunk 1]
```diff
@@ -1103,7 +1104,7 @@
 		var/turf/target_turf = get_turf(target) //Does the nice effects first so whatever happens to what's about to get clapped doesn't affect it
 		var/obj/effect/temp_visual/kinetic_blast/K = new /obj/effect/temp_visual/kinetic_blast(target_turf)
 		K.color = color
-		playsound(loc, 'sound/effects/gravhit.ogg', 80, TRUE) //Mainly this sound
+		playsound(loc, 'sound/effects/powerhammerhit.ogg', 80, TRUE) //Mainly this sound
```
Comment (Moltijoe) [inline code/game/objects/items/twohanded.dm]:
```suggestion
		playsound(loc, 'sound/effects/powerhammerhit.ogg', 80, FALSE) //Mainly this sound
```

> by that do you mean the explosion sound that plays alongside the current one?

i mean do this

**Review by Moltijoe — APPROVED** [review]
shrimple changes

**Comment by Moltijoe** [comment]
what's with the electruichum.ogg?

**Comment by solwashere** [comment]
Right yeah, that's a leftover from what I was working on (a charge sound effect) but scrapped due to what I was trying to achieve not really being possible from what I was told

**Comment by Moltijoe** [comment]
> Right yeah, that's a leftover from what I was working on (a charge sound effect) but scrapped due to what I was trying to achieve not really being possible from what I was told

if you don't plan to use it, it's probably best to just remove it

File: sound/effects/electrichum.ogg
Comment (ReddicusDragon) [inline sound/effects/electrichum.ogg:1]: DELETE THIS IF IT IS UNUSED please and thank you sol

**Review by ReddicusDragon — CHANGES_REQUESTED** [review]

**Review by ReddicusDragon — APPROVED** [review]

**Comment by ToasterBiome** [comment]
Copywritten sounds... 💀

**Comment by Moltijoe** [comment]
> Copywritten sounds... 💀

no they aren't what are you talking about, they're completely original, or from creative commons, or under fair use

**Comment by neeshacark** [comment]
even if they are copyrighted... that's never stopped ya'll before. we have shittons of stuff like that

**Review by ToasterBiome — APPROVED** [review]

**Pull request merged**